ciao-cli/main.go

```diff
@@ -1031,7 +1038,12 @@ func main() {
 		if *imageName == "" {
 			fatalf("Missing required -image-name parameter")
 		}
-		createTenantImage(*identityUser, *identityPassword, id, *imageName)
+		if *imageVisibility != "private" && *imageVisibility != "public" {
+			fatalf("image-visibility can be either 'private' or 'public'")
+		}
+		createTenantImage(*identityUser, *identityPassword, id, *imageName,
+			*imagePath, *containerFormat, *diskFormat, *imageVisibility, *imageTags,
+			*minDiskGigabytes, *minRAMMegabytes, *protected)
```

Comment:
At that point it would make sense to create a `type imageOptions struct` and pass a pointer to `createTenantImage()`.